src/Components/Samples/BlazorUnitedApp/BlazorUnitedApp.csproj

```diff
@@ -4,11 +4,13 @@
     <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
     <IsShippingPackage>false</IsShippingPackage>
     <Nullable>enable</Nullable>
+    <RazorLangVersion>8.0</RazorLangVersion>
```

Comment:
This is required for the `rendermode=...` temporary API to work, because we have overridden `AddComponentParameter`.

Reply:
Removed